specs/fault-proof/stage-one/anchor-state-registry.md

```diff
@@ -194,8 +393,38 @@ Determines if a game is a Retired Game.
 
 Determines if a game is a Proper Game.
 
-- MUST return `true` if and only if `isGameRegistered(game)` and `isGameRespected(game)` are both
-  `true` and `isGameBlacklisted(game)` and `isGameRetired(game)` are both `false`.
+- MUST return `true` if and only if `isGameRegistered(game)` is `true` and
+  `isGameBlacklisted(game)` and `isGameRetired(game)` are both `false`.
+
+### isGameResolved
+
+Determines if a game is a Resolved Game.
+
+- MUST return `true` if and only if the game has resolved a result in favor of either the
+  Challenger or the Defender as determined by the `FaultDisputeGame.status()` function.
+
+### isGameAirgapped
+
+Determines if a game is an Airgapped Game.
+
+- MUST return `true` if and only if the game has resolved a result more than the airgap delay
+  seconds ago as defined by the `disputeGameFinalityDelaySeconds` variable in the `OptimismPortal`
+  contract.
+
+### isGameFinalized
```

Comment:
As noted above, these two functions will always return the same result as each other.